android/app/src/main/java/com/example/composelambda/pages/DetailPage.kt

```diff
@@ -99,37 +127,41 @@ fun BuildDetailPage(actions: Actions) {
-                        "The former vice president is also stretching his leads in Nevada and Georgia, though each state remains too close to call. He leads Trump by more than 22,000 votes in Nevada and is ahead by more than 7,000 in Georgia. The count will be complicated in Pennsylvania by tens of thousands of provisional ballots and many others that require extra care for reasons that include damage, legibility, signature issues or other defects.\n" +
-                        "The President cannot reach 270 electoral votes without winning both Pennsylvania and Georgia, and at least one of the other outstanding states. Biden can get over the top by winning Pennsylvania on its own or by taking both Nevada and Arizona. The challenger currently leads the President by 253 to 213 electoral votes, CNN projects.\n" +
-                        "Trump is cutting the Democrat's lead in Arizona, which is down to fewer than 30,000 votes with 94% reported, but it is not clear whether his margins are sufficiently wide to overtake his rival with 235,000 votes still to be counted.\n" +
-                        "Biden's late-night appearance in Wilmington, Delaware, struck a contrast to Trump's grievance-filled speech the night before, in which he flung false claims that the election is being stolen and vowed to fight on in the courts. Claiming a mandate to ease the nation's angry divides, the former vice president vowed that life in America under a Biden administration would be less angst-ridden and acrimonious, and on the worst day yet for Covid-19 infections -- more than 125,000 -- he offered condolences to those who have lost loved ones.\n" +
-                        "\"We have to remember: The purpose of our politics isn't total, unrelenting, unending warfare. No. The purpose of our politics, the work of the nation, isn't to fan the flames of conflict -- but to solve problems,\" Biden said. \"To guarantee justice. To give everybody a fair shot. To improve the lives of our people.\"\n",
+                    text = detailContent[1],
                     style = MaterialTheme.typography.body1
                         .copy(
                             fontFamily = FontFamily.Monospace,
                             letterSpacing = 0.12.em,
-                            fontSize = (MaterialTheme.typography.body1.fontSize.value + delta).sp,
+                            fontSize = (MaterialTheme.typography.body1.fontSize.value + fontSizeDelta).sp,
                         )
 
                 )
-                Spacer(modifier = Modifier.preferredHeightIn(16.dp))
+                Spacer(modifier = Modifier.height(16.dp))
             }
         }
     )
 }
+
+@Composable
+fun NewsViewModel.parserNewsDetail(newsType: NewsType): Array<String> {
+    return when (newsType) {
+        NewsType.BreakingNews -> {
+            val content by breakingNewsDetail.collectAsState(initial = BreakingNews.default)
```

Comment:
From the Flutter side, there is "fetch", then populate the state with data.

Maybe for Android, it could be done in the same way, because, sometimes, time2time, the default page will be shown and won't be updated until opening it next time.